src/inmanta/server/services/resourceservice.py

```diff
@@ -251,7 +251,6 @@ async def mark_deployed(
         version: int,
         filter: Callable[[ResourceIdStr], bool] = lambda x: True,
         connection: ConnectionMaybeInTransaction = ConnectionNotInTransaction(),
-        only_update_from_states: Optional[Sequence[const.ResourceState]] = None,
```

Comment:
This was no longer used?

Reply:
There was only one call to `mark_deployed` on core and it did not use that argument. By the end of this epic we are expecting to completely remove `mark_deployed` from the codebase